test/extended/deployments/util.go

```diff
@@ -33,6 +33,7 @@ import (
 
 	appsapiv1 "github.com/openshift/api/apps/v1"
 	appsapi "github.com/openshift/origin/pkg/apps/apis/apps"
+	appsinternalutil "github.com/openshift/origin/pkg/apps/controller/util"
```

Comment:
extended tests should use external clients; a followup maybe

Reply:
definitely follow up

Reply:
at least this split will make it clear where we use internal helpers